yjit_codegen.c

```diff
@@ -300,7 +313,7 @@ yjit_gen_exit(jitstate_t *jit, ctx_t *ctx, codeblock_t *cb)
     mov(cb, member_opnd(REG_CFP, rb_control_frame_t, pc), RAX);
 
     // Accumulate stats about interpreter exits
-#if RUBY_DEBUG
+#if RUBY_DEBUG || YJIT_STATS
```

Comment:
This should probably be #if YJIT_STATS, no reason to make it an or, unless I'm missing something.

Reply:
Good catch, I fixed it.